docs/app/documentation/component-docs/datetime-picker/examples/datetime-format-example/datetime-format-example.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Injectable } from '@angular/core';
+import { DateTimeFormatParser } from '../../../../../../../library/src/lib/datetime-picker/format/datetime-parser';
+
+@Injectable()
+export class DateTimeFormatExample extends DateTimeFormatParser {
+
+    public parse(value: string): Date {
+
+        value = value.replace(/\//g, '-').replace(/ /g, '-');
+        const values: number[] = value.split('-').map(Number);
+
+        return new Date(Number(values[0]), values[1] - 1, values[2], values[3], values[4], values[5]);
+    }
+
+    public format(date: Date): string {
+        const getAtLeastTwoDigits = function(num: number): string { return num < 10 ? '0' + num : String(num) };
+        return date.getFullYear() + '-' +
+            getAtLeastTwoDigits(date.getMonth() + 1) + '-' +
+            date.getDate() + ' ' +
+            getAtLeastTwoDigits(date.getHours()) + '/' +
+            getAtLeastTwoDigits(date.getMinutes()) + '/' +
+            getAtLeastTwoDigits(date.getSeconds());
```

Comment:
I think this would make more sense: 
```
getAtLeastTwoDigits(date.getHours()) + 'h' +
getAtLeastTwoDigits(date.getMinutes()) + 'm' +
getAtLeastTwoDigits(date.getSeconds() + 's');
```